src/libtorch.cc

```diff
@@ -1508,6 +1531,9 @@ ModelInstanceState::Execute(
     // enable/disable inference mode - supersedes NoGradGuard
     torch::InferenceMode infer_guard(model_state_->EnabledInferenceMode());
 
+    // enable/disable cudnn
+    at::globalContext().setUserEnabledCuDNN(model_state_->EnabledCudnn());
```

Comment:
I wonder if this should be called per inference execution? Or would it make sense to make this call once within `ModelState::LoadModel`

Reply:
Filed a ticket for following up on this.